docs/src/main/sphinx/release/release-419.md

```diff
@@ -0,0 +1,41 @@
+# Release 419 (24 May 2023)
+
+## General
+
+* Add the {func}`array_histogram` function to find the number of occurrences of
+  the unique elements in an array. ({issue}`14725 `)
+* Fix JSON serialization failure when using string columns with values larger
```

Comment:
How does it manifest? Is it a client-coordinator failure? A worker-worker communication failure?

I would just say "Fix failure when processing string columns with values larger than 5 MB", or something to that effect.

cc @hashhar @wendigo

Reply:
Actually it can be both @martint